spec/models/involved_civilian_spec.rb

```diff
@@ -53,5 +53,23 @@
       civilian.received_force_location = ['Head']
       expect(civilian.valid?).to be false
     end
+
+    it 'returns true if general_info.on_k12_campus is true and k12_type is not nil' do
```

Comment:
I know we haven't been good about this before, but I think that this spec is getting big enough that it should be broken down into smaller `describe` blocks. I'm thinking something like:

```ruby
describe InvolvedCivilian, type: :model do
  describe 'valid?' do
    describe '[Mental status]' do
      ...
    end

    describe '[Weapons and force]' do
      ...
    end

    describe '[Civilians on a K12 campus]' do
       ...
    end
  end
end
```